COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class CoordinateSystem //координатна система: містить Ліст, який зберігає всі точки, метод Add, який вносить ці точки в ліст, DisplayAllPoints, який друкує всі наявні точки
+    {
+        List<Point> listOfPoints = new List<Point>();
+        List<PointsDistance> pointsDistances = new List<PointsDistance>();
+
+        //Point pointClass = new Point();
+
+        public void Add(Point point)
+        {
+            foreach (Point point1 in listOfPoints)
+            {
+                //Point point2 = point;
+                PointsDistance distance = new PointsDistance(point, point1, point.DistanceTo(point1));
+                pointsDistances.Add(distance);
+            }
+            listOfPoints.Add(point);
+        }
+
+        public void DisplayAllPoints()
+        {
+            int count = 1;
+
+            foreach (Point i in listOfPoints)
+            {
+                Console.Write($"point {count++} : {i};\n");
+            }
+        }
+
+        public PointsDistance MaxDistance()
+        {
+            PointsDistance pointsDistance2 = null;
```

Comment:
why not just `pointsDistance`?

Reply:
i don't get what you want... i need to use Class (type) but not its name... (mb i just misscomunicate of what you try to say to me)

Reply:
why this `PointsDistance pointsDistance2 = null;` but not this `PointsDistance pointsDistance = null;`